src/libs/ReportUtils.js

```diff
@@ -460,6 +449,23 @@ function isConciergeChatReport(report) {
     return lodashGet(report, 'participantAccountIDs', []).length === 1 && Number(report.participantAccountIDs[0]) === CONST.ACCOUNT_ID.CONCIERGE && !isChatThread(report);
 }
 
+/**
+ * Check if report is a DM and personal detail of participant is optimistic data
+ * @param {String} report
+ * @returns {Boolean}
+ */
+function shouldDisableDetailPage(report) {
+    const participantAccountIDs = lodashGet(report, 'participantAccountIDs', []);
+
+    if (isChatRoom(report) || isPolicyExpenseChat(report) || isChatThread(report)) {
```

Comment:
I feel like it would be nice to explain whyyy we're skipping the check for only these specific report types 🤔

Reply:
Aah ok @dukenv0307 so `isChatRoom` doesn't check if this is a DM, it checks `isUserCreatedPolicyRoom(report) || isDefaultRoom(report)` - so this now seems like it makes sense 👍 

I feel like `isChatRoom` isn't very clear, since I would expect there to be many more chat reports 😅